libraries/resource_providers/api1200/synergy/server_profile_template_provider.rb

```diff
@@ -0,0 +1,20 @@
+# Copyright 2020 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+module OneviewCookbook
+  module API1200
+    module Synergy
+      # Server Profile Template API1200 Synergy provider
+      class ServerProfileTemplateProvider < OneviewCookbook::API1000::C7000::ServerProfileTemplateProvider
```

Comment:
i think it should inherit from API1200 keeping it sync with other API's implementation. Though in our case, there won't be any failure but in future, if we are adding in 1200 API, it should get inherit in Synergy API 1200 as well.

Reply:
Fixed